bin/circle/install-circle-predependencies.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+# Add google-chrome source
+wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | sudo apt-key add -
+sudo sh -c 'echo "deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google.list'
```

Comment:
nit: for parallelism's sake, it would be nice if this line followed the same pattern as line 10 below: `echo "deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main" | sudo tee /etc/apt/sources.list.d/google.list`.